.github/workflows/test.yml

```diff
@@ -487,6 +487,36 @@ jobs:
         with:
           projectBaseDir: schema/
 
+  test-schema-testing:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "1.22"
+          cache: true
+          cache-dependency-path: schema/testing/go.sum
+      - uses: technote-space/get-diff-action@v6.1.2
+        id: git_diff
+        with:
+          PATTERNS: |
+            schema/testing/**/*.go
+            schema/testing/go.mod
+            schema/testing/go.sum
+      - name: tests
+        if: env.GIT_DIFF
+        run: |
+          cd schema
+          go test -mod=readonly -timeout 30m -coverprofile=coverage.out -covermode=atomic ./...
+      - name: sonarcloud
```

Comment:
is the goal to have it in a separate project? If so you are missing the sonar property file in schema/testing

Reply:
It needs go 1.22 so probably

Reply:
Just added the sonar property file

Reply:
@julienrbrt looks like we need some setup the sonar side, getting this error in CI:
```
ERROR: Could not find a default branch for project with key 'cosmos-sdk-schema-testing'. Make sure project exists.
```

Reply:
I can add the project yes

Reply:
https://sonarcloud.io/project/configuration?id=cosmos-sdk-schema-testing